samples/RewriteSample/Startup.cs

```diff
@@ -30,9 +31,16 @@ public static void Main(string[] args)
             var host = new WebHostBuilder()
                 .UseKestrel(options =>
                 {
-                    options.UseHttps("testCert.pfx", "testPassword");
+                    options.Listen(IPAddress.Loopback, 5000, listenOptions =>
+                    {
+                        listenOptions.UseConnectionLogging();
```

Comment:
You don't need connection logging in this sample. You can just do `options.Listen(IPAddress.Loopback, 5000);`